public/js/controllers/address_controller.js

```diff
@@ -227,7 +227,8 @@ export default class extends Controller {
 
   bindElements () {
     this.flowBoxes = this.flowTarget.querySelectorAll('input')
-    this.pageSizeOptions = this.pagesizeTarget.querySelectorAll('option')
+    // pagesizeTarget is not available for dummy addresses
+    this.pageSizeOptions = this.data.get('isDummy') === 'false' ? this.pagesizeTarget.querySelectorAll('option') : []
```

Comment:
The comment is really what you are after, so I suggest checking for the target directly rather than adding a data element:

```js
this.pageSizeOptions = this.hasPagesizeTarget ? this.pagesizeTarget.querySelectorAll('option') : []
```

https://stimulusjs.org/reference/targets